ext/standard/config.m4

```diff
@@ -455,7 +455,7 @@ PHP_NEW_EXTENSION(standard, array.c base64.c basic_functions.c browscap.c crc32.
                             http_fopen_wrapper.c php_fopen_wrapper.c credits.c css.c \
                             var_unserializer.c ftok.c sha1.c user_filters.c uuencode.c \
                             filters.c proc_open.c streamsfuncs.c http.c password.c \
-                            random.c,,,
+                            random.c hrtime.c,,,
```

Comment:
@jpauli That's talking about CPU time for benchmarking, that's not the use case I have. While benchmarking is another possible use case for `hrtime()`, the reason I created this PR and am interested in `hrtime()` is for event loops. I'm not even interested in a nanosecond resolution, it just happens to be provided. Milliseconds would be enough for my use case.